src/app-host/live-reload-client.js

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Based in part on code from Vogue (https://github.com/andrewdavey/vogue)
+
+var url = require('url');
+
+var canRefreshEventName = 'lr-can-refresh';
+var refreshFileEventName = 'lr-refresh-file';
+var fullReloadEventName = 'lr-full-reload';
```

Comment:
You can define all of this constants in an enumeration
{code}
var EventsName = {
    CAN_REFRESH: 'lr-can-refresh',
    REFRESH_FILE: 'lr-refresh-file',
    FULL_RELOAD: 'lr-full-reload'
};
{code}